calculators/referendum_supporters_calculator.rb

```diff
@@ -6,11 +6,19 @@ def initialize(candidates: [], ballot_measures: [], committees: [])
   end
 
   def fetch
+    # UNION Schedle E with the 24-Hour IEs from 496.
     expenditures = ActiveRecord::Base.connection.execute(<<-SQL)
       SELECT "Filer_ID", "Filer_NamL", "Bal_Name", "Sup_Opp_Cd",
         SUM("Amount") AS "Total_Amount"
-      FROM "efile_COAK_2016_E-Expenditure"
-      WHERE "Bal_Name" IS NOT NULL
+      FROM (
+        SELECT "Filer_ID", "Filer_NamL", "Bal_Name", "Sup_Opp_Cd", "Amount"
+        FROM "efile_COAK_2016_E-Expenditure"
+        WHERE "Bal_Name" IS NOT NULL
+        UNION
+        SELECT "Filer_ID"::varchar, "Filer_NamL", "Bal_Name", "Sup_Opp_Cd", "Amount"
+        FROM "efile_COAK_2016_496"
```

Comment:
The data that is there does not seem to support this. The only value in FormType is 'F496'